polkadot/runtime/westend/src/lib.rs

```diff
@@ -1180,6 +1181,13 @@ impl InstanceFilter<RuntimeCall> for ProxyType {
 					RuntimeCall::Registrar(..) |
 					RuntimeCall::Slots(..)
 			),
+			ProxyType::ParaRegistration => matches!(
+				c,
+				RuntimeCall::Registrar(paras_registrar::Call::reserve { .. }) |
+					RuntimeCall::Registrar(paras_registrar::Call::register { .. }) |
+					RuntimeCall::Utility(..) |
```

Comment:
Please match directly for batch etc calls and not all of the pallet.

Reply:
Added! :)